nazi_zombies_portable/egg-nazi-zombies--portable.json

```diff
@@ -0,0 +1,82 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-06-17T20:41:26-04:00",
+    "name": "Nazi Zombies: Portable",
+    "author": "jordanadamsbusiness@outlook.com",
+    "description": "NZ:P (Nazi Zombies: Portable) is based off of the FTEQW (Fore Thought Engine Quake World). This egg is specifically made for use with NZ:P (just downloads assets and QuakeC from NZ:P repositories) but there is nothing stopping you from using it as a general FTEQW egg given the NZ:P fork of FTEQW only added fog to the client (and added SDL which caused issues with arguments being eaten preventing dedicated flag from being read) which is why upstream FTEQW is used.",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:debian": "ghcr.io\/parkervcp\/yolks:debian"
+    },
+    "file_denylist": [],
+    "startup": ".\/{{SERVER_BINARY}} -dedicated +sv_port {{SERVER_PORT}} +sv_port_tcp {{SERVER_PORT}} +com_protocolname {{PROTOCOL}} +sv_mintic {{TICKRATE}} +map {{MAP}} {{GAME_ARGS}}",
```

Comment:
sv_port_tcp  should always be set to 0.0.0.0 so it binds to all interfaces

Reply:
ummm... it is a port variable not an address variable? I could try to change it to `0.0.0.0:{{SERVER_PORT}}` and test it but doesn't seem like it'd fit?

Reply:
I read this wrong. This is fine.